concepts/teams-changenotifications-teammembership.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Get change notifications for any change in Teams membership using Microsoft Graph"
+description: "Get change notifications for any change in Teams membership using Microsoft Graph"
+author: "Abhishek Anand"
+localization_priority: Priority
+ms.prod: "microsoft-teams"
+ms.custom: scenarios:getting-started
+---
+
+# Get change notifications for change in teams membership using Microsoft Graph
+
+Change notifications enable you to subscribe to changes (create, update, and delete) to [messages](/graph/api/resources/chatMessage?preserve-view=true) in a [channel](/graph/api/resources/channel?preserve-view=true) or [chat](/graph/api/resources/chat?preserve-view=true). Change notifications provide a low latency model by allowing you to maintain a [subscription](/graph/api/resources/webhooks?preserve-view=true). You can also get the resource data in the notifications and therefore avoid calling the API to get the payload.
+
+>**Note:** The maximum time a subscription can last is 60 minutes; however, subscriptions can be renewed until the caller has permissions to access to resource.
+
+
+
+### Subscribe to changes in membership of any team across tenant
+
+To get change notifications for membership change in any team in tenant, subscribe to `teams/getAllMembers`. This resource supports [including resource data](webhooks-with-resource-data.md) in the notification.
```

Comment:
One way in which this subscription is different in the others is, multiple resources of this type can be created/deleted in one go. Example: from teams or groups UI, one can add 5 members to a group in one go, but the subscriber is still going to get 5 notifications, one for every user added, not one notifiation with info of all 5 users added. We have to mention this too.